### DISCUSSION TITLE
Python SDF Library

### DISCUSSION THREAD

**Discussion by erny123** [epochpic/epoch#146] [Q&A]
Hi Epoch team,

I was wondering if you will ever have a manual or maybe short list of function descriptions for the sdf python library?

There's some functions such as plot_path,plot_rays, plot_levels, etc.  that would be interesting to know what they do. 

Thanks,
Ernesto

**Comment by TomGoffrey** [comment]
I think the short answer is no.

We will be re-hosting our old documentation for EPOCH again shortly (although possibly in a different format to the previous version), and that includes some documentation for visualising the output from EPOCH. However, SDF is an extensive library used by several other codes, so the EPOCH related documentation will only discuss a subset of the full functionality.

**Reply by erny123** [reply]
Got it. 
Thanks for the reply!
